dependencies/dependencies.ini

```diff
@@ -1,5 +1,9 @@
 [runtime]
-# 'python' and 'python-dependencies' keys expects a value in the Python requirements file format
+# 'python' and 'python-dependencies' keys expect a value in the Python requirements file format
 #  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format
 python=torch==1.7.1
        wheel
+
+[device]
+# These dependencies require a CPU. If no CPU is available, the package will not be installed.
```

Comment:
No, I really meant to write _CPU_ here. The flag is not strictly required in this `depencies.ini` file since the fallback is the assumption that a CPU suffices. However, I thought that it would be good to add it anyway since the file (being at the top level) might be used as a reference by some developers. For the same reason (template), I added the comment.